htscodecs/rANS_static4x16pr.c

```diff
@@ -1607,7 +1607,6 @@ unsigned char *rans_uncompress_to_4x16(unsigned char *in,  unsigned int in_size,
 		goto err;
 	}
     } else {
-	tmp1 = NULL;
```

Comment:
Not that I disagree with the change (and have already accepted it in unroll32 branch), but I'm curious what's actually undefined here?

The C standard I checked states for things like memcpy that reading beyond the bounds of an array is (reasonably) undefined.  It doesn't state that a NULL pointer is undefined.  This feels a bit like code obscurity in order to silence a false positive.  Generally it's considered good behaviour to reset a pointer to NULL when we know we shouldn't be accessing it.  So removing this may be best done with a coment.